src/constants/style-related.ts

```diff
@@ -13,5 +13,5 @@ export const SIZES = ['small', 'normal', 'large'] as const;
 export const HEADING_SIZES = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'] as const;
 export const CHECKBOX_VIEWBOX = [0, 0, 10, 10] as const;
 
-export type ContextType = typeof CONTEXTS[number];
-export type SizeType = typeof SIZES[number];
+export type Context = typeof CONTEXTS[number];
+export type Size = typeof SIZES[number];
```

Comment:
```
typeof SIZES[number];
```
this is not the most obvious TS construction. Might be makes sense to add a comment here.